rules/false_positives/datadog.yara

```diff
@@ -0,0 +1,12 @@
+rule ddtrace_rules_json : override {
+  meta:
+    description = "appsec/rules.json"
+    linux_multi_persist = "high"
```

Comment:
this should be "ignore" or something equivalent.

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).